src/App.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div id="app" class="bg-gray-50 h-screen dark:bg-gray-900" >
```

Comment:
It's not a good idea to use a viewport to set the main height of the app, because are some incompatibilities in Safari iOS and MacOS (Webkit).